src/shell.js

```diff
@@ -235,11 +235,13 @@ if (ENVIRONMENT_IS_NODE) {
   // EXPORT_ES6 + ENVIRONMENT_IS_NODE always requires use of import.meta.url,
   // since there's no way getting the current absolute path of the module when
   // support for that is not available.
-  scriptDirectory = require('url').fileURLToPath(new URL('./', import.meta.url)); // includes trailing slash
+  scriptDirectory = nodePath.dirname(require('url').fileURLToPath(import.meta.url));
 #else
-  scriptDirectory = __dirname + '/';
+  scriptDirectory = __dirname;
 #endif
 
+scriptDirectory += '/' // scriptDirectory is expected to end in a "/"
```

Comment:
Can you just add `+ '/'` to the end of both lines above rather than adding a second statement here?  I know it looks like code duplication but it helps on code size.